src/budget/index.html

```diff
@@ -121,11 +121,11 @@ <h2 class="h3">Funding requests</h2>
                     block__border-right">
             <div class="content-l content-l__main">
                 <section class="content-l_col content-l_col-1-2">
-                    <h1 class="header-slug">
+                    <h2 class="header-slug">
```

Comment:
This looks like it would be a style change potentially. Is `/budget/` the same on flapjack.demo vs local?